src/Middleware/Diagnostics.Abstractions/src/IExceptionHandlerEndpointFeature.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+
+namespace Microsoft.AspNetCore.Diagnostics
+{
+    /// <summary>
+    /// Represents an exception handler with the original Endpoint and path of the request.
+    /// </summary>
+    public interface IExceptionHandlerEndpointFeature : IExceptionHandlerPathFeature
```

Comment:
I'm not sure how I feel about this feature. Are we gonna have another one that derives from this? Why aren't we using DIMs here?

Reply:
What is DIM? I couldn't find a relevant meaning by googling it.

Reply:
Default interface methods

Reply:
DIMs only work for methods, not properties, right?

Reply:
Nevermind, there are DIM properties as well.

Reply:
I think auto properties don't work (since they are backed by implicit fields) but Property => null; Should work AFAIK

Reply:
How about obsoleting the `IExceptionHandlerPathFeature` and add all properties with DIM on `IExceptionHandlerFeature`?

Reply:
I'm not sure, is that a breaking change? What happens to the Path property?

Reply:
Path property would be also added to `IExceptionHandlerFeature` with DIM like the Endpoint and RouteValues.

Reply:
Anyone using IExceptionHandlerPathFeature would have to remove it. I don't know if it's worth the break for people.